File: scrimage-filters/src/main/scala/com/sksamuel/scrimage/filter/GothamFilter.scala
package com.sksamuel.scrimage.filter

import java.util

import com.sksamuel.scrimage.PipelineFilter

object GothamFilter extends PipelineFilter(util.Arrays.asList(
  HSBFilter(0, -0.85, 0.2),
  ColorizeFilter(34, 43, 109, 25),
  GammaFilter(0.5),
  ContrastFilter(1.4)
))

File: scrimage-filters/src/main/java/com/sksamuel/scrimage/filter/GothamFilter.java
package com.sksamuel.scrimage.filter;

import java.util.Arrays;

public class GothamFilter extends PipelineFilter {

    public GothamFilter() {
        super(Arrays.asList(
                new HSBFilter(0, -0.85f, 0.2f),
                new ColorizeFilter(34, 43, 109, 25),
                new GammaFilter(0.5),
                new ContrastFilter(1.4)
        ));
    }
}
